api/helpers/functions.ts

```diff
@@ -197,6 +198,9 @@ export async function timedQuery(
         request.method === 'POST' ? request.body : request.query
     );
 
+    console.log(cachedResponse, 'in case it needed to be stringified')
```

Comment:
console is only  incase it was supposed to be stringified so I dont get [object, object]